lib/Demangling/NodePrinter.cpp

```diff
@@ -21,6 +21,7 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <stack>
```

Comment:
Unused?

Reply:
Yeah, from before I realized that swift-demangle could show me the exact structure of the names I was interested in and was writing graph search stuff to try to explore the structure. Thanks for catching this.